charts/px-backup/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+description: A Helm chart for installing PX-Backup with PX-Central on Kubernetes and Openshift
+home: https://portworx.com/
+icon: https://raw.githubusercontent.com/portworx/helm/master/doc/media/px-logo.png
+keywords:
```

Comment:
Add `apiVersion: v1` to make helm lint pass

``` helm lint .
==> Linting .
[ERROR] Chart.yaml: apiVersion is required. The value must be either "v1" or "v2"

Error: 1 chart(s) linted, 1 chart(s) failed
```

Reply:
Added.
==> Linting charts/px-backup/

1 chart(s) linted, 0 chart(s) failed